<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsFile_ExplorerClass</class>
 <widget class="QMainWindow" name="QtWidgetsFile_ExplorerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>393</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>QFile_Explorer_Test</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1051</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="Input_Wnd">
       <property name="text">
        <string>Enter file path to monitor :  </string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Action_Btn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>1051</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="Input_Wnd_2">
       <property name="text">
        <string>Enter path tosave Logs :</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Action_Btn_2">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="Output_Wnd">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>1051</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1092</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWidgetsFile_Explorer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Action_Btn</sender>
   <signal>clicked(bool)</signal>
   <receiver>Action_Btn</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1069</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>1069</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Input_Wnd</sender>
   <signal>textEdited(QString)</signal>
   <receiver>Input_Wnd</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Input_Wnd_2</sender>
   <signal>textEdited(QString)</signal>
   <receiver>Input_Wnd_2</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Action_Btn_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>Action_Btn_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1038</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>1042</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
